plugins/spectate/spectate.cpp

```diff
@@ -110,6 +110,10 @@ DFhackCExport command_result plugin_load_data (color_ostream &out) {
         pause_lock->unlock();
         enable_auto_unpause(out, unpause_enabled);
     }
+    if (enabled) {
+        std::vector<std::string> params;
+        spectate(out, params);
```

Comment:
this will also cause the config to be saved. is that desirable, or should you factor out the status report into a separate function and call that instead?

Reply:
The save is logically pointless, but it shouldn't hurt. Do you wish me to factor it out?

Reply:
I'll just do it, there are other changes I would like to throw in